src/test/javascript/spec/component/assessment-shared/assessment-header.component.spec.ts

```diff
@@ -328,4 +328,82 @@ describe('AssessmentHeaderComponent', () => {
         fixture.detectChanges();
         expect(sendAssessmentEvent).toHaveBeenCalledTimes(0);
     });
+
+    it('should save assessment on control and s', () => {
+        component.isLoading = false;
+        fixture.detectChanges();
+        const eventMock = new KeyboardEvent('keydown', { ctrlKey: true, key: 's' });
+        const saveButton = fixture.debugElement.query(By.css('#save')).nativeElement;
+        const spyOnControlAndS = jest.spyOn(component, 'saveOnControlAndS');
+        const saveSpy = jest.spyOn(saveButton, 'click');
+
+        expect(saveButton).toBeTruthy();
```

Comment:
```suggestion
        expect(saveButton).not.toBeNull();
```
As per the guidelines `toBeTruthy/Falsy` shouldn’t be used as that can lead to accidentally wrong assertions (e.g., `0` is defined but also falsy).